.dockerignore

```diff
@@ -0,0 +1,4 @@
+frontend/node_modules
+frontend/public
+backend/node_modules
+backend/dist
```

Comment:
Can you add a new blank line at the end of the file? 😉 Otherwise it looks good!

Reply:
yup, great catch!

Reply:
done